### PR TITLE
Feature/home general

### PR DESCRIPTION
**Pull request by lacf95**
Completed sidebar for homepage

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
     @user = User.find(session[:user_id])
-    @pool = EtherClient.new.pool
+    general_ethermine
+  end
+
+  private
+
+  def general_ethermine
+    ether = EtherClient.new
+    stats = ether.pool.statistics
```
Comment (PedroTerriquez) [inline app/controllers/users_controller.rb]: Why this isn't instance variable?

Comment (lacf95) [reply app/controllers/users_controller.rb]: Why should it?

**Comment by lacf95** [comment]
I added the instance variables of each end point to avoid the view to call those, instead the controller does it.

**Comment by PedroTerriquez** [comment]
Great job my friend!

**Pull request merged**